vortex-array/src/array/varbinview/compute.rs

```diff
@@ -45,3 +52,22 @@ impl SliceFn for VarBinViewArray {
         .into_array())
     }
 }
+
+impl CompareFn for VarBinViewArray {
+    fn compare(&self, other: &Array, operator: Operator) -> VortexResult<Array> {
+        let lhs = self.clone().into_canonical()?.into_arrow();
```

Comment:
Can this logic be implemented in the compare entry point function?

Reply:
Not if you're comparing other types, right? Ideally if you compare compressed arrays, they should stay that way for comparison.

Reply:
Yeah, so check if the array implements compare or not, and fallback to this implementation.